acl/pom.xml

```diff
@@ -13,7 +13,7 @@
     <parent>
         <groupId>org.apache.rocketmq</groupId>
         <artifactId>rocketmq-all</artifactId>
-        <version>5.0.1-SNAPSHOT</version>
+        <version>5.1.1-SNAPSHOT</version>
```

Comment:
Is this should be 5.1.0?

Reply:
It's the version after release. See commit for more information